<?xml version="1.0"?>
<launch>
    <rosparam file="$(find plane_part_stewart_driver)/config/calibration.yaml" />

    <include file="$(find plane_part_moveit_config)/launch/demo.launch">
        <!-- must use gui -->
        <arg name="use_gui" value="true" />
    </include>

    <node name="stewart_driver_client" pkg="plane_part_stewart_driver"
        type="stewart_driver_client.py" />

    <node name="stewart_driver" pkg="plane_part_stewart_driver" type="stewart_bringup.py" />

</launch>